### ISSUE TITLE
All article timestamps appear hours in the future 

### ISSUE THREAD

**Issue by Trevortds** [Tunous/SwipeNews-Issues#146]
Using Feedbin

All articles, no matter what the source, appear with a timestamp several hours ahead of correct. Brand new ones read "in 2 hours", whereas articles that show something "2 minutes ago" are actually about two hours old.

**Comment by Trevortds** [comment]
![screenshot_20180927-091343](https://user-images.githubusercontent.com/6858009/46153306-ddd1a680-c240-11e8-8781-fd60a947db6c.png)

**Comment by Tunous** [comment]
What is your system language and timezone?

**Comment by Trevortds** [comment]
Within the app? Language is "default" and I can't find any time zone setting.

**Comment by Tunous** [comment]
Yeah, I meant these in Android. It's probably an issue with timezone calculations. Will have to investigate when I get more time.
